DrawerKit/DrawerKit/Internal API/PresentationController+Setup.swift

```diff
@@ -1,5 +1,28 @@
 import UIKit
 
+extension PresentationController {
+    func setupPresentationContainerView() {
+        guard self.presentationContainerView == nil else { return }
+
+        let presentationContainerView = PresentationContainerView()
+        presentationContainerView.touchDelegate = self
+        presentationContainerView.backgroundColor = .clear
+        presentationContainerView.frame = containerView?.superview?.frame ?? .zero
+        presentationContainerView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+
+        if let containerView = containerView {
+            containerView.superview?.addSubview(presentationContainerView)
+            presentationContainerView.addSubview(containerView)
+        }
+
+        self.presentationContainerView = presentationContainerView
+    }
+
+    func removePresentationContainerView() {
+        presentationContainerView.removeFromSuperview()
```

Comment:
Don't you have to restore `containerView` as a subview of its original superview?

Reply:
this happens when dismissal finishes so container view would be removed anyway by UIKit after that.

Reply:
The, perhaps add a comment explaining that? It would prevent others from getting confused in the same way I did.

Reply:
I added comment

Reply:
Ok, except for the grammar :) It should be

> // the containerView is removed by UIKit when dismissal ends
> // so it's not necessary to restore it in its original superview

Reply:
@wltrup can you please add it using "suggestion" functionality?